File: langs/cowsay.kt
fun main(a:Array<String>){
    val m=if(a.isEmpty())"Hello, World!" else a.joinToString(" ")
    print(" "+"_".repeat(m.length+2)+"\n< $m >\n "+"-".repeat(m.length+2)+"\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n")
}

File: langs/cowsay.rs
fn main(){
    let a:Vec<String>=std::env::args().skip(1).collect();
    let m=if a.is_empty(){"Hello, World!".to_string()}else{a.join(" ")};
    print!(" {}\n< {} >\n {}\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n","_".repeat(m.len()+2),m,"-".repeat(m.len()+2));
}
